packages/plans-grid-next/src/hooks/data-store/use-plan-billing-description.tsx

```diff
@@ -32,6 +33,7 @@ export default function usePlanBillingDescription( {
 	isMonthlyPlan,
 	coupon,
 	useCheckPlanAvailabilityForPurchase,
+	reflectStorageSelectionInPlanPrices,
```

Comment:
I didn't get to testing the PR today, but can do so tomorrow.

One question: the reason we are not grabbing this from context is that it sits under `/hooks/data-store/...`, which we may move to data-stores, hence not able to query the context - is this correct? 🤔

Reply:
> One question: the reason we are not grabbing this from context is that it sits under /hooks/data-store/..., which we may move to data-stores, hence not able to query the context - is this correct? 🤔

Good question -- this was actually an oversight on my part 😅 To me, it makes more sense to keep usage of context consistent within the `plans-grid-next` package, if possible, to avoid confusion until we migrate to data-stores.

I'll go through and double check the files where it makes sense to reference context.

Reply:
[5f19efd](https://github.com/Automattic/wp-calypso/pull/96520/commits/5f19efd3438f9f2ad0c10e099d9b10c357180dae). Easy to revert if we change our minds